systemlayer/executionrequests.go

```diff
@@ -8,32 +8,58 @@ import (
 	pilosa "github.com/featurebasedb/featurebase/v3"
 )
 
+const (
+	keepLastRequests int = 2000
+	truncateTextAt   int = 4096
```

Comment:
these can be untyped constants which will give us a bit more flexibility. (just remove "int") https://go.dev/blog/constants